TYPES.md

```diff
@@ -100,6 +103,70 @@ Parameters are :
 | secondary |
 | `hasKey(<key>)(state)` | returns `true` if the given `<key>` is present | |
 
+## simple
+### factory
+```es6
+import { simple } from 'k-redux-factory'
+
+export default simple(/* options */)
+```
+
+or
+
+```es6
+import factory from 'k-redux-factory'
+
+export default factory({ type: 'simpleObject', /* other options */ })
+```
+
+There are multiple factories signatures, take you favorite between:
+ - `simple(middlewares)(options)` : this is a `simple` factory with middlewares
+ - `simple(options)` : this is a `simple` factory without middlewares
+ - `simple.bool(options)` : this is a `simple boolean` factory initialize to `false`
+ - `simple.string(options)` : this is a `simple string` factory initialize to `''`
+ - `simple.array(options)` : this is a `simple array` factory initialize to `[]`
```

Comment:
we should add `simple.object`

Reply:
by default, `simple()` init with empty object.
you think it's important to add this shorter ?

Reply:
For me yes, it's better to be explicit about this since the name doesn't contains 'object' anymore.

Reply:
Yes, simple can do a simple.object per default, but we have to add it explicitly

Reply:
@guillaumecrespel can you update the readme please ?